### PR TITLE
Vincent dev 0.0.2

### PR DESCRIPTION
**Pull request by lutianzhou001**

### CODE REVIEW

File: api/GetAssetByHashLEInHex.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# GetAssetByHashInHex
```
Comment (vang1ong7ang) [inline api/GetAssetByHashLEInHex.md]: fixname

File: api/GetAssetByHashLEInJSON.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# GetAssetByHashInJSON
```
Comment (vang1ong7ang) [inline api/GetAssetByHashLEInJSON.md]: fixit

File: api/GetUTXOByHashInHex.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+}
+```
+
+## Output
+
+```json
+{
+    "type": "string",
+    "pattern": "[0-9a-f]+",
+    "description": "output data encoded in hex format"
```
Comment (vang1ong7ang) [inline api/GetUTXOByHashInHex.md]: utxo data encoded in hex format

File: api/GetAssetByHashLEInHex.md (hunk 1 above)
Comment (lutianzhou001) [reply api/GetAssetByHashLEInHex.md]: done

File: api/GetAssetByHashLEInJSON.md (hunk 2 above)
Comment (lutianzhou001) [reply api/GetAssetByHashLEInJSON.md]: done

File: api/GetUTXOByHashInHex.md (hunk 3 above)
Comment (lutianzhou001) [reply api/GetUTXOByHashInHex.md]: done

**Comment by vang1ong7ang** [comment]
add description of parameter: `Hash` to GetStorageByHashHeightInHex

**Pull request merged**